config/helm/eventing-auth-manager/templates/deployment.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "eventing-auth-manager.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/component: eventing-auth-manager.kyma-project.io
+    app.kubernetes.io/created-by: eventing-auth-manager
+    app.kubernetes.io/part-of: eventing-auth-manager
+  {{- include "eventing-auth-manager.labels" . | nindent 4 }}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "eventing-auth-manager.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/component: eventing-auth-manager.kyma-project.io
+    app.kubernetes.io/created-by: eventing-auth-manager
+    app.kubernetes.io/part-of: eventing-auth-manager
+    control-plane: controller-manager
+  {{- include "eventing-auth-manager.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicas }}
+  selector:
+    matchLabels:
+      app.kubernetes.io/component: eventing-auth-manager.kyma-project.io
+      control-plane: controller-manager
+    {{- include "eventing-auth-manager.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/component: eventing-auth-manager.kyma-project.io
+        control-plane: controller-manager
+      {{- include "eventing-auth-manager.selectorLabels" . | nindent 8 }}
+      annotations:
+        kubectl.kubernetes.io/default-container: manager
+    spec:
+      containers:
+      - args:
+        - --health-probe-bind-address=:8081
+        - --metrics-bind-address=127.0.0.1:8080
+        - --leader-elect
```

Comment:
What the purpose of `--leader-elect` arg?

Reply:
this argument used by the `ctl.Manager` for leader-election [here](https://github.com/muralov/eventing-auth-manager/blob/24168790d55ce225802480a9b5de4ac44aab8c4a/cmd/main.go#L65)